common/src/main/java/com/wynntils/mc/utils/ItemUtils.java

```diff
@@ -26,7 +25,7 @@
 
 public class ItemUtils {
 
-    private static final Pattern ITEM_RARITY_PATTERN =
+    public static final Pattern ITEM_RARITY_PATTERN =
```

Comment:
Actually not too sure about this, should this be public or should this be re-defined in the other file? Should we perhaps have a file containing these patterns?

Reply:
honestly i don't think this pattern should exist in this file at all - it's used in a method that removes wynn flavor text, which should go in `WynnItemUtils` (and also be renamed to something more clear). it's fine to keep the pattern public once it's moved imo

Reply:
Moved the method. About the pattern file, let's open a discussion and see what others think.